src/layout/Popup.js

```diff
@@ -50,11 +50,7 @@
         HTMLWidget.prototype.update.apply(this, arguments);
         var context = this;        
 
-        if (this.popupState()) {
-            element.style("display", "block");
-        } else {
-            element.style("display", "none");
-        }
+        this.visible(this.popupState());
```

Comment:
@GordonSmith Yes, I was going to bring this up to you once the whole thing was ready for your review, I'm still working on things from J and Matts comments.  Should I make an issue for the Popup widget change for the PR to reference?